src/clients/nfs_server.rb

```diff
@@ -225,8 +228,10 @@ def check_packages
     # @param [Hash] options command options
     # @return whether successful
     def NfsServerStartHandler(options)
-      options = deep_copy(options)
-      return false if NfsServer.start
+      if NfsServer.start
+        CommandLine.Print(_("NFS server already running."))
+        return false
```

Comment:
BTW I'm not sure if starting NFS server is an error if it already is running... but as that was the previous implementation lets probably keep that...

Reply:
yeah, I try to do minimal changes in behavior if someone already has working script. But in general I agree with you and probably main reason is to skip write operation.